src/t_list.c

```diff
@@ -803,8 +802,7 @@ void blockingPopGenericCommand(client *c, int where) {
     for (j = 1; j < c->argc-1; j++) {
         o = lookupKeyWrite(c->db,c->argv[j]);
         if (o != NULL) {
-            if (o->type != OBJ_LIST) {
-                addReply(c,shared.wrongtypeerr);
+            if (checkType(c,o,OBJ_LIST)) {
```

Comment:
so you avoided getting this out and fixing the unnecessary indentation in order not to cause conflicts by shifting so much code, right?
i certainly agree.

Reply:
Yeah, that is why I didn't touch this one.